src/animation/PropertyBinding.js

```diff
@@ -543,12 +543,7 @@ THREE.PropertyBinding.parseTrackName = function( trackName ) {
 	//	  .bone[Armature.DEF_cog].position
 	// created and tested via https://regex101.com/#javascript
 
-	var index = trackName.lastIndexOf( '/' );
-	var directoryName = trackName.slice( 0, Math.max( 0, index ) );
-
-	var re = /^([^.[\]]+)?(?:\.([^.[\]]+)(?:\[([^[\]]+)\])?)?\.(?:([^.[\]]+)(?:\[([^[\]]+)\])?)$/;
-	re.lastIndex = index + 1;
-
+	var re = /^((?:[\w]+\/)*)([\w-\d]+)?(?:\.([\w]+)(?:\[([^[\]]+)\])?)?\.([\w]+)(?:\[([^[\]]+)\])?$/;
```

Comment:
Since `\w` does not match internationalized strings, I wonder whether the rules different in that case. Also, it's the same as `[\w]`, so there's no need for the brackets.

Why use `\w` at all instead of just going with the "is-not-delimiter" expression `[^.[\]]`?